<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>akniga.org downloader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <property name="horizontalSpacing">
         <number>6</number>
        </property>
        <item row="3" column="0" colspan="3">
         <widget class="QPushButton" name="buttonDownload">
          <property name="text">
           <string>Запустить скачивание</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="buttonOpenPath">
          <property name="text">
           <string>Выбрать</string>
          </property>
          <property name="icon">
           <iconset theme="Open">
            <normaloff>../../</normaloff>../../</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelURL">
          <property name="text">
           <string>URL</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QLineEdit" name="lineURL"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="linePath"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelPath">
          <property name="text">
           <string>Каталог</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QMdiArea" name="mdiArea"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Окно</string>
    </property>
    <addaction name="actionCascade"/>
    <addaction name="actionTile"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="text">
    <string>Каскадом</string>
   </property>
   <property name="toolTip">
    <string>Каскадом</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="text">
    <string>Плиткой</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Каскадом</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Плиткой</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOpenPath</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onButtonOpenPathClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonDownload</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onButtonDownloadClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCascade</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>cascadeSubWindows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTile</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>tileSubWindows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onButtonOpenPathClick()</slot>
  <slot>onButtonDownloadClick()</slot>
 </slots>
</ui>
